handlers/cookies/handler.go

```diff
@@ -0,0 +1,18 @@
+package cookies
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/ONSdigital/dp-frontend-models/model"
+	"github.com/ONSdigital/dp-frontend-renderer/render"
+)
+
+//Handler ...
+func Handler(w http.ResponseWriter, req *http.Request) {
+	var page model.Page
+
+	fmt.Printf("%+v\n", page)
```

Comment:
If this isn't necessary anymore for debugging etc. then we ought to remove this

Reply:
👍